manim/scene/scene.py

```diff
@@ -727,6 +731,32 @@ def get_moving_and_static_mobjects(self, animations):
         )
         return all_moving_mobject_families, static_mobjects
 
+    @contextmanager
+    def changing_scene_speed_to(
+        self,
+        speedinfo: Dict[float, float],
+        retain_final_speed=True,
```

Comment:
By default, the context manager should only change things inside of its context. Retaining the final speed by default goes against the idea of a context manager -- this could actually be made into an argument of whether a context manager is even the right tool for this job.

```suggestion
        retain_final_speed=False,
```